internal/storage/read_test.go

```diff
@@ -92,6 +92,7 @@ func Test_extractServersFromBytes(t *testing.T) {
 					"privatevpn": {"version": 1, "timestamp": 0},
 					"protonvpn": {"version": 1, "timestamp": 0},
 					"purevpn": {"version": 1, "timestamp": 0},
+					"slickvpn": {"version": 1, "timestamp": 0},
```

Comment:
@qdm12 change this test file so it's not required to update it when adding a provider

Reply:
Done in beb4550774f00fc7922e07bb6d742ea00d0ebf37